packages/legend-application-query/src/application/LegendQueryApplicationConfig.ts

```diff
@@ -112,6 +115,7 @@ export class LegendQueryApplicationConfig extends LegendApplicationConfig {
   readonly engineQueryServerUrl?: string | undefined;
   readonly depotServerUrl: string;
   readonly studioApplicationUrl: string;
+  readonly taxonomyApplicationUrl: string;
```

Comment:
can we make this property optional. This should not be required to run stand alone